CMakeLists.txt

```diff
@@ -45,8 +45,8 @@ else()
     add_compile_options(-Wno-gnu-zero-variadic-macro-arguments)
   endif()
   if(SANITIZERS)
-    set(SAN_FLAGS -fsanitize=shift -fsanitize=integer-divide-by-zero
-                  -fsanitize=unreachable -fsanitize=vla-bound
+    set(SAN_FLAGS -fsanitize=shift -fsanitize=integer-divide-by-zero -fsanitize=leak
+                  -fsanitize=undefined -fsanitize=unreachable -fsanitize=vla-bound
```

Comment:
Note that `-fsanitize=undefined` is broken down into the `-fsanitize` flags we are currently using. (I'm not sure why we're not using `-fsanitize=undefined`.)

Point is, either we keep our current litany of flags, or we scrap all of them in favour of just `-fsanitize=undefined`. (I'd prefer the latter, if able.)